x-pack/plugins/observability_solution/investigate_app/public/components/investigate_view/index.tsx

```diff
@@ -201,162 +192,80 @@ function InvestigateViewWithUser({ user }: { user: AuthenticatedUser }) {
   }
 
   return (
-    <MiniMapContextProvider container={scrollableContainer}>
-      <EuiFlexGroup direction="row" className={containerClassName}>
-        <EuiFlexItem grow className={scrollContainerClassName}>
-          <EuiFlexGroup
-            direction="column"
-            gutterSize="s"
-            justifyContent="flexEnd"
-            ref={(element) => {
-              // the type for `EuiFlexGroup.ref` is not correct
-              const asHtmlElement = element as unknown as HTMLDivElement;
-              stickToBottomRef(asHtmlElement);
-              setScrollableContainer(asHtmlElement);
-            }}
-          >
-            <EuiFlexItem className={searchBarContainerClassName}>
-              <InvestigateSearchBar
-                kuery={displayedKuery}
-                rangeFrom={range.from}
-                rangeTo={range.to}
-                onQuerySubmit={async ({ kuery: nextKuery, dateRange: nextDateRange }) => {
-                  const nextTimeRange = {
-                    from: datemath.parse(nextDateRange.from)!.toISOString(),
-                    to: datemath.parse(nextDateRange.to)!.toISOString(),
-                  };
-                  await setGlobalParameters({
+    <EuiFlexGroup direction="row" className={containerClassName}>
+      <EuiFlexItem grow className={scrollContainerClassName}>
+        <EuiFlexGroup direction="column" gutterSize="s" justifyContent="flexEnd">
+          <EuiFlexItem grow={false} key={AddWidgetMode.Esql}>
+            <EuiButton data-test-subj="investigateAppInvestigateViewWithUserAddAnObservationChartButton">
+              {i18n.translate(
+                'xpack.investigateApp.investigateViewWithUser.addAnObservationChartButtonLabel',
+                { defaultMessage: 'Add an observation chart' }
+              )}
+            </EuiButton>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFlexItem>
+
+      <EuiFlexItem grow={false} className={sideBarClassName}>
+        <EuiFlexGroup direction="column" gutterSize="m">
+          <EuiFlexItem className={gridContainerClassName} grow={false}>
+            <InvestigateWidgetGrid
```

Comment:
I moved the InvestigateWidgetGrid to the sidebar, so that added notes appear on the sidebar.

TODO: Once we add the other widgets (ESQL, embeddable) in the UI, we need to rethink the InvestigateWidgetGrid, because now all widgets (included notes) appear in one common timeline. That's not what we want in the mock up design